### DISCUSSION TITLE
开启绕过中国大陆ip之后的系统日志出现垃圾域名

### DISCUSSION THREAD

**Discussion by c71an** [vernesong/OpenClash#2925] [Q&A]
![捕获](https://user-images.githubusercontent.com/15607041/210963632-69ff97bd-72cc-4d44-bd50-bc96efeb0fb2.PNG)
### 这是DNS被污染了吗？关闭该功能之后不再复现。如何处理？

Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: started, version 2.86 cachesize 150
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: DNS service limited to local subnets
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: compile time options: IPv6 GNU-getopt no-DBus UBus no-i18n no-IDN DHCP DHCPv6 no-Lua TFTP conntrack ipset auth cryptohash DNSSEC no-ID loop-detect inotify dumpfile
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: UBus support enabled: connected to system bus
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq-dhcp[1]: DHCP, IP range 192.168.1.100 -- 192.168.1.249, lease time 12h
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 127.0.0.1#7874
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 0-100.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 0-6.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 00.net
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 000.link
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 00000.host
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 00042.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 00058.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 0006266.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 0007.net
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 000714.xyz
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 000dn.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 000e.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001133.app
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001daima.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001en.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001jm.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001job.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001kd.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001pp.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001tech.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 001town.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 002lzj.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 0033.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 0037wan.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 00394.net
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 003store.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 004218.com
Fri Jan  6 15:57:50 2023 daemon.info dnsmasq[1]: using nameserver 114.114.114.114#53 for domain 004678.com

**Comment by pengid8** [comment]
我的也出现了

**Comment by vernesong** [comment]
这是加载列表提示
